cosmwasm/submit-proposal.js

```diff
@@ -135,12 +130,8 @@ const instantiate = async (client, wallet, config, options) => {
         const chainName = chain.toLowerCase();
 
         const { contractName, instantiate2, predictOnly, fetchCodeId } = options;
-        const {
-            axelar: {
-                contracts: { [contractName]: contractConfig },
-            },
-            chains: { [chainName]: chainConfig },
-        } = config;
+        const contractConfig = getAmplifierContractConfig(config, contractName);
+        const chainConfig = getChainConfig(config, chainName);
```

Comment:
Can we remove the multiple chain support next?

Reply:
was planning to prioritize it like this:
1. subcommands for non-governance scripts
2. migrate script through admin account
3. remove multiple chain support

let me know if you think I should re-prioritize

Reply:
I think removing multiple chain support, and standardizing chain specific vs generic config reading will make the others easier to implement and review